concent_api/core/validation.py

```diff
@@ -218,3 +225,81 @@ def validate_list_task_to_compute_ids(subtask_results_accepted_list):
     for task_to_compute in subtask_results_accepted_list:
         subtask_ids.append(task_to_compute.subtask_id + ':' + task_to_compute.task_id)
     return len(subtask_ids) == len(set(subtask_ids))
+
+
+def validate_file_transfer_token(file_transfer_token):
```

Comment:
Add typings.

Reply:
If You mean type annotation then its done :)